devservices/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+# Ignored by docker compose, used by devservices
+x-sentry-service-config:
+  version: 0.1
+  service_name: relay
+  dependencies:
+    redis:
+      description: "redis"
+    kafka:
+      description: "kafka"
+  modes:
+    default: [redis, kafka]
+
+services:
+  redis:
+    image: "redis:6.2.14-alpine"
+    healthcheck:
+      test: redis-cli ping
+  kafka:
+    image: "confluentinc/cp-kafka:7.6.1"
```

Comment:
@hubertdeng123  how the redis and kafka versions will be kept in-sync with Sentry repo? 
Is it all manual open/merge PRs operation?

Reply:
We are planning on making a shared definitions for kafka and redis to be used across Sentry services to ensure versions and other settings are kept in-sync

Reply:
Indeed, not available initially but it is something we are planning to add as we implement the logic for this.